js/application.js

```diff
@@ -126,194 +209,507 @@ $(document).ready(function(){
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+      </div>\
+      <div class = "row" id="grid_6">\
+        <div id="column1" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column2" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column3" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column4" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+      </div>\
+      <div class = "row" id="grid_7">\
+        <div id="column1" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column2" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column3" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column4" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+      </div>\
+      <div class = "row" id="grid_8">\
+        <div id="column1" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column2" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column3" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column4" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
```

Comment:
```
<div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>
```

These are all duplicates. You need to write a function (under 10 line codes, with a loop) that will generate these grid.